.travis.yml

```diff
@@ -30,7 +30,6 @@ script:
 
   # show the daos that are indexed
   - ./scripts/checkDaos.sh
-  - echo -en 'travis_fold:end:script.4\\r'
```

Comment:
this should not get removed!

Reply:
that's weird, must have been a weird merge? I didn't do that